src/main/resources/db/migration/V2018.07.26__Initialize_database.sql

```diff
@@ -0,0 +1,65 @@
+CREATE TABLE problem_set(
+    id SERIAL PRIMARY KEY,
+    edit_url VARCHAR (255) UNIQUE NOT NULL,
```

Comment:
can we change 'url' to 'code', since we won't store entire urls here, but rather just the code part, without the host and all that.

Reply:
Right, nice catch